modules/Sources/Features/Home/Home.swift

```diff
@@ -232,23 +233,26 @@ public struct Home: Reducer {
                 }
                 state.synchronizerState = latestState
                 state.synchronizerStatusSnapshot = snapshot
-                state.shieldedBalance = latestState.shieldedBalance.redacted
-                state.transparentBalance = latestState.transparentBalance.redacted
+                state.shieldedBalance = (latestState.accountBalance?.saplingBalance.spendableValue ?? .zero) +
+                    (latestState.accountBalance?.orchardBalance.spendableValue ?? .zero)
+                state.transparentBalance = latestState.accountBalance?.unshielded ?? .zero
+                state.totalBalance = (state.synchronizerState.accountBalance?.saplingBalance.total() ?? .zero) +
+                    (latestState.accountBalance?.orchardBalance.spendableValue ?? .zero) +
```

Comment:
I'd just store `latestState.accountBalance?.orchardBalance.spendableValue` into their local constants to make this more readable.